website/views.py

```diff
@@ -492,7 +501,7 @@ def process_issue(self, user, obj, created, domain, tokenauth=False, score=3):
 
 class IssueCreate(IssueBaseCreate, CreateView):
     model = Issue
-    fields = ["url", "description", "screenshot", "domain", "label"]
```

Comment:
can keep this

Reply:
this is not needed cause we have custom form validation to vaildate screenshots.
Issue create wont work cause name=screenshot is changed to screenshots  in html
and we will not be supporting screenshot from now on so it will give error during validation cause no html field is renamed to screenshots